### DISCUSSION TITLE
Should we release 0.70.4?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#38] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/43

---

Current Release: [0.70.3](https://github.com/facebook/react-native/releases/tag/v0.70.3)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### Issues we are aware of

1. 

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/2f6b2127d933094f864523749d13cfbb140b5b63 (needs react-native-codegen bump)
2. https://github.com/facebook/react-native/commit/74fda1070266df13e1b58680a670dde3acf9d205
3. https://github.com/facebook/react-native/commit/70cc27c901aeb447910e30ac3ceac85990d3c16d
4. https://github.com/facebook/react-native/commit/a24c8946e065ca89048e574abd7c2dc0434a350b

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
potentially: https://github.com/facebook/react-native/commit/70cc27c901aeb447910e30ac3ceac85990d3c16d

see https://github.com/reactwg/react-native-new-architecture/discussions/81#discussioncomment-3895344

**Comment by kelset** [comment]
https://github.com/facebook/react-native/pull/34738

**Reply by kelset** [reply]
(will need a codegen bump)

**Comment by krystofwoldrich** [comment]
Maybe https://github.com/facebook/react-native/pull/34895

**Comment by kelset** [comment]
https://github.com/facebook/react-native/pull/35022

**Comment by jacobp100** [comment]
https://github.com/facebook/react-native/pull/34974 (flex gap)? Credits should include @intergalacticspacehighway @NickGerleman and @jacobp100

**Reply by spyshower** [reply]
For 0.71 release, yeah count me in.

**Reply by kelset** [reply]
yeah this is too big of a feature to drop into a patch release. This will be available in 0.71 - will open the road to 0.71 discussion later today actually

**Comment by yoonsy** [comment]
https://github.com/facebook/react-native/pull/34215

**Reply by kelset** [reply]
we're already done the cherry picking and 0.70.4 is rolling out right now, you'll have to request this again on the 0.70.5 discussion

**Reply by dmytrorykun** [reply]
@yoonsy please submit this pick here https://github.com/reactwg/react-native-releases/discussions/43
